3_load_conjur_policies.sh

```diff
@@ -39,9 +37,7 @@ if [[ "${DEPLOY_MASTER_CLUSTER}" == "true" ]]; then
   $cli_with_timeout "exec $conjur_cli_pod -- \
     bash -c \"
       CONJUR_ADMIN_PASSWORD=${CONJUR_ADMIN_PASSWORD} \
-      TEST_APP_NAMESPACE_NAME=${TEST_APP_NAMESPACE_NAME} \
-      POSTGRES_USERNAME=${POSTGRES_USERNAME} \
-      POSTGRES_PASSWORD=${POSTGRES_PASSWORD} \
```

Comment:
why did we remove the postgres variables? aren't they needed?

Reply:
demo

Reply:
removed

Reply:
nope, they were for demo